docs/using-latest-daily.md

```diff
@@ -68,7 +62,7 @@ Alternatively, if you are using Visual Studio, you can instead create a new Blaz
 
 ## [Optional] Using scripts to install the latest .NET Aspire build from release branches
 
-If you want to install the latest build from the main branch, then you shouldn't follow the next steps, and instead check out: [Add necessary NuGet feeds](#add-necessary-nuget-feeds).
+If you want to install the latest build from the main branch, then you shouldn't follow the next steps, and instead check out: [Run the workload installation script](#run-the-workload-installation-script).
```

Comment:
This section is now kind of redundant since above we say:

"The [workload installation script](./../eng/installLatestFromReleaseBranch.ps1)) will install the latest .NET Aspire workload from the release branch, but it can also install latest from main (latest nightly build) if the `-FromMain` flag is used (`--fromMain` on Linux/macOS)."

What do you think about combining these sections now?

Reply:
Fair point, should've spotted that. I will fix this. Actually, latest from release branch is really no longer useful any longer as we now only produce stable builds from release branches (and stable builds aren't supported by the script). I may just remove this section for now and then add support in the future for installing the latest released version, and update these docs for that.